MatrixSDK/Space/MXSpaceService.swift

```diff
@@ -87,6 +108,114 @@ public class MXSpaceService: NSObject {
         let room = self.session.room(withRoomId: spaceId)
         return room?.toSpace()
     }
+        
+    /// Get the space children informations of a given space from the server.
+    /// - Parameters:
+    ///   - spaceId: The room id of the queried space.
+    ///   - parameters: Space children request parameters.
+    ///   - completion: A closure called when the operation completes.
+    /// - Returns: a `MXHTTPOperation` instance.
+    @discardableResult
+    public func getSpaceChildrenForSpace(withId spaceId: String,
+                                         parameters: MXSpaceChildrenRequestParameters?,
+                                         completion: @escaping (MXResponse<MXSpaceChildrenSummary>) -> Void) -> MXHTTPOperation {
+        return self.session.matrixRestClient.getSpaceChildrenForSpace(withId: spaceId, parameters: parameters) { (response) in
+            switch response {
+            case .success(let spaceChildrenResponse):
+                guard let rooms = spaceChildrenResponse.rooms else {
```

Comment:
Should we have a processing queue?
I heard that we can expect crazy spaces hierarchies.